superset/config.py

```diff
@@ -114,6 +114,9 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 VERSION_SHA_LENGTH = 8
 VERSION_SHA = _try_json_readsha(VERSION_INFO_FILE, VERSION_SHA_LENGTH)
 
+# default viz used in chart explorer
+DEFAULT_EXPLORER_VIZ = 'table'
```

Comment:
nit: I would prefer to call this `DEFAULT_VIZ_TYPE`

Reply:
okay sounds good I can change that.